src/metabase/actions.clj

```diff
@@ -37,7 +38,7 @@
 
 (defmulti normalize-action-arg-map
   "Normalize the `arg-map` passed to [[perform-action!]] for a specific `action`."
-  {:arglists '([action arg-map])}
+  {:arglists '([action arg-map]), :added "0.44.0"}
```

Comment:
Forgot to add this stuff earlier. But all driver methods should have `^:added` metadata so we know what version we introduced them in.

I know this stuff probably won't actually ship as part of 44, but we can go update these numbers down the road if we need to -- so 44 is mostly a placeholder.